File: composeApp/src/androidMain/kotlin/studio/nxtech/fujubank/splash/SplashConfig.kt
package studio.nxtech.fujubank.splash

/**
 * 起動スプラッシュの最低表示時間。
 *
 * `SessionStore.bootstrap()` がこれより早く完了してもロゴを最低この時間は見せる。
 * iOS 側の `SplashConfig.minDuration` (秒) と値を揃えること。
 *
 * Figma でアニメーション尺が指定された場合は両プラットフォームで同時に更新する。
 */
object SplashConfig {
    const val MIN_DURATION_MS: Long = 2000L
}

File: iosApp/iosApp/Features/Splash/SplashConfig.swift
import Foundation

/// 起動スプラッシュの最低表示時間。
///
/// `SessionStore.bootstrap()` がこれより早く完了してもロゴを最低この時間は見せる。
/// Android 側 `SplashConfig.MIN_DURATION_MS` (ミリ秒) と値を揃えること。
enum SplashConfig {
    static let minDuration: TimeInterval = 2.0
}
